misk-feature/src/main/kotlin/misk/feature/AppScopedDynamicConfigResolver.kt

```diff
@@ -0,0 +1,13 @@
+package misk.feature
+
+/**
+ * Abstraction for dynamically retrieving dynamic config overrides, scoped to the calling app.
+ */
+interface AppScopedDynamicConfigResolver<T : ValidatableConfig<T>> {
+  /**
+   * Resolves the named configuration scoped to the calling application.
+   * If the underlying feature is not found, or its JSON representation cannot be interpreted,
```

Comment:
this seems very specific to the `Real` impl. why do you even have an interface? would we ever need a fake resolver. seems like `FakeFeatureFlags` is enough.

Reply:
Yeah, no need considering this is nothing more than a shim